hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionWrapper.java

```diff
@@ -170,4 +170,15 @@
    *   all compacted store files that belong to this region
    */
   long getMaxCompactedStoreFileRefCount();
+
+  /**
+   * @return the number of reads on memstore
+   */
+  long getMemstoreReadRequestsCount();
```

Comment:
... this if for the Region?  Can I ask for the number of reads on memstore on a Store?

Reply:
I just followed a model where currently we have accouting at table level and region level. So following the same path - We added per region per store metric and this one is nothing but a region level aggregation. This will tell on the region how many reads are from memstore. (across all stores).